### PR TITLE
add boilerplate integration tests for sharepoint

### PR DESCRIPTION
**Pull request by ryanfkeepers**
## Type of change

- [x] :robot: Test

## Issue(s)

* #1506

## Test Plan

- [x] :green_heart: E2E

### CODE REVIEW

File: src/internal/tester/resource_owners.go [hunk 1]
```diff
@@ -42,8 +55,29 @@ func LoadTestM365UserID(t *testing.T) string {
 	return cfg[TestCfgLoadTestUserID]
 }
 
-// expects cfg value to be a string representing an array like:
-// "['foo@example.com','bar@example.com']"
+// expects cfg value to be a string representing an array such as:
+// ["site1\,uuid","site2\,uuid"]
+// the delimeter must be a |.
```
Comment (ashmrtn) [inline src/internal/tester/resource_owners.go:60]: this comment does not seem to match the array given on the line above

File: src/pkg/repository/repository_load_test.go [hunk 2]
```diff
@@ -552,3 +560,101 @@ func (suite *RepositoryIndividualLoadTestOneDriveSuite) TestOneDrive() {
...
+	bsel.Include(bsel.Sites(suite.sitesUnderTest))
+	sel := bsel.Selector
+
+	runLoadTest(
+		suite.T(),
+		ctx,
+		suite.repo,
+		"all_sites", "share_point",
+		suite.sitesUnderTest,
+		sel, sel, // same selection for backup and restore
```
Comment (ashmrtn) [inline src/pkg/repository/repository_load_test.go:611]: is restore defined for SharePoint right now?

**Review by ashmrtn — APPROVED** [review]

File: src/pkg/repository/repository_load_test.go (hunk 2 above)
Comment (ryanfkeepers) [reply src/pkg/repository/repository_load_test.go:611]: Nope, but if you look a little up from here you'll see that these tests are currently skipped.

File: src/internal/tester/resource_owners.go (hunk 1 above)
Comment (ryanfkeepers) [reply src/internal/tester/resource_owners.go:60]: Yeah... we have some things to figure out about resource owner lookup with sites.  I can come back and correct his when we do.

**Pull request merged**